### DISCUSSION TITLE
主题色与搜索栏

### DISCUSSION THREAD

**Discussion by Cauliflower12315** [the1812/Bilibili-Evolved#2798] [提问 / Q&A]
①新版没找到在哪调主题色，能教下吗
②新版好像在动态和视频里没有自定义顶栏中的透明填充，而且希望搜索栏的图标改回放大镜，且不要显示那行“暂无搜索记录”，而且新版bilibili动态、消息之类的图标设计还不错，希望可以进行保留

**Comment by Geeyun-JY3** [comment]
1. 设置面板 > 通用设置 > 主题颜色
![image](https://user-images.githubusercontent.com/49396421/147749786-4a2bbe3b-0512-4c7b-99cf-86e6b2a5f77d.png)
2. `自定义顶栏` 的透明填充需要有横幅，本来就不会在动态和视频页生效
https://github.com/the1812/Bilibili-Evolved/blob/master/doc/features.md#样式
    > - 在顶部横幅存在时, 使用透明填充

    V1 设置项说明
![image](https://user-images.githubusercontent.com/49396421/147750491-9b146308-34dc-42e0-89a3-94638e8403a3.png)

    为啥不是放大镜了见：https://github.com/the1812/Bilibili-Evolved/discussions/2195#discussioncomment-1379985

    自定义顶栏可以自定义入口顺序，新版顶栏图标只有右边的动态消息那堆，要搞图标的话我建议全都上图标，但官方目前没有给左边部分入口的图标，就得自己画或找。

**Reply by the1812** [reply]
~~回复写一半被你抢发了~~

加图标不太能保证一致性, 自定义顶栏上不仅仅是自带的那些, 还可以加插件注入其他按钮, 不可能要求每个开发者弄这个还要画个图标

**Reply by Geeyun-JY3** [reply]
那行“暂无搜索记录”你还是可以回应一下的嘛🙈

所以我觉得加图标有些麻烦。

**Reply by Cauliflower12315** [reply]
主要是动态和视频里面的搜索框就变得不是下划线那样，就不太好看，v1就跟首页搜索框一致，更好看。将顶栏动态展开成一块一块的是非常天才的设计，大赞！

**Reply by the1812** [reply]
> 那行“暂无搜索记录”你还是可以回应一下的嘛🙈

我觉得没这个弹出来的话, 不好区分是搜索建议崩了还是只是没数据

> 主要是动态和视频里面的搜索框就变得不是下划线那样，就不太好看，v1就跟首页搜索框一致，更好看。

没明白你指的是啥, 要不截点图看看?

**Reply by Cauliflower12315** [reply]
> > 那行“暂无搜索记录”你还是可以回应一下的嘛🙈
> 
> 我觉得没这个弹出来的话, 不好区分是搜索建议崩了还是只是没数据
> 
> > 主要是动态和视频里面的搜索框就变得不是下划线那样，就不太好看，v1就跟首页搜索框一致，更好看。
> 
> 没明白你指的是啥, 要不截点图看看?
![v2](https://user-images.githubusercontent.com/96857365/147760426-c1c697ec-d642-46c8-935d-70bd4ee42478.jpg)
![v1](https://user-images.githubusercontent.com/96857365/147760432-bf76b683-a105-4a09-8b50-b76e395a3fe1.jpg)

**Reply by Cauliflower12315** [reply]
上面一张是v2，下面是v1

**Reply by Geeyun-JY3** [reply]
> > 主要是动态和视频里面的搜索框就变得不是下划线那样，就不太好看，v1就跟首页搜索框一致，更好看。
> 
> 没明白你指的是啥, 要不截点图看看?

你是不是忘了：https://github.com/the1812/Bilibili-Evolved/discussions/2195#discussioncomment-1221945

**Reply by the1812** [reply]
我主要是不明白"跟首页搜索框一致", 自定义顶栏开了的话, 首页/视频/动态不都是长得一样的吗?

**Reply by Cauliflower12315** [reply]
> 我主要是不明白"跟首页搜索框一致", 自定义顶栏开了的话, 首页/视频/动态不都是长得一样的吗?

v2好像就是首页跟另外两个不一样，在开了透明填充的情况下
